### DISCUSSION TITLE
Alexa does not find Sonoff

### DISCUSSION THREAD

**Discussion by Kev-2003** [arendst/Tasmota#13318] [Q&A] [answered]
Hello

I flashed my Sonoff with Tasmota yesterday and wanted to integrate it with Alexa again. But Alexa no longer finds the Sonoff. The Sonoff can be reached via the website. 
Does anyone know what the problem is?

Thanks for the help.

**Comment by barbudor** [comment]
Hi
You don't give much details
Did you read this part of the docs https://tasmota.github.io/docs/Alexa/ ?

**Comment by Kev-2003** [comment]
I have an Echo Dot 3 Gen. and 2 Echo Dot 4 Gen. My red one is a FRITZ!Box7490.
The Sonoff is a Sonoff S20. 

I hope this helps
![Screenshot 2021-10-10 133746](https://user-images.githubusercontent.com/92253463/136693935-33d1db1f-3734-4334-8a5c-f3a6eaf8baae.png)
.

**Reply by barbudor** [reply]
Please update to latest development version
I'm not an Alexa specialist but I think some things where changed since release of 9.5.0

**Comment by s-hadinger** [comment, marked as answer]
And provide logs with `weblog 4` level. 

Are you sure you didn't block multicast between Echo and Tasmota?
